change/@microsoft-teams-js-cd4a4578-9d98-4c5e-be23-8d5bed594601.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added `meeting.registerMeetingReactionReceivedHandler`",
+  "packageName": "@microsoft/teams-js",
+  "email": "joeugeni@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
not really a deal breaker, but I would probably mark it as 'minor'

Reply:
updated the dependentChangeType to _minor_